web/content/docs/benchmarks/python-bc/hertz-contact/hertz-contact.md

```diff
@@ -0,0 +1,100 @@
++++
```

Comment:
Could you rename the `.md` to a `hertz-contact.pandoc`, please. See https://github.com/ufz/ogs/pull/2205 for reason...